cmd/adapter/adapter.go

```diff
@@ -295,6 +299,50 @@ func (cmd *PrometheusAdapter) addResourceMetricsAPI(promClient prom.Client, stop
 	return nil
 }
 
+func setupMetricsPort() {
```

Comment:
see comment below about existing metrics - if there's already some handler, we should reuse it

Reply:
See comment above